javascript/javascript3/week1/giphy/giphy.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="giphy.css">
+    <title>Giphy</title>
+</head>
+<body>
+
+    <div class="input-keyword">
+        <h1>GIFs searcher</h1>
+        <div>
+            <label for="inputKeyword"></label>
```

Comment:
It is recommended to keep only one naming convention. Either camelCase or underscore-with-hyphens.